index.html

```diff
@@ -16,22 +16,27 @@
         <script src="js/vendor/modernizr-2.8.3.min.js"></script>
     </head>
     <body>
-        <div class="tab">
-            <a href="/open-data-progress-report/alt-version.html">Click here to see an alternative layout.</a>
-        </div>
```

Comment:
So I know this was ugly but it served a purpose to compare the two layouts and the feedback we got was in favor of the alternative "stacked layout". We either need to keep this or we should really just go ahead and adopt the columns and rows layout.